setup-jest.ts

```diff
@@ -3,4 +3,6 @@ import { Crypto } from '@peculiar/webcrypto';
 
 enableFetchMocks();
 
-global.crypto = new Crypto();
+if (!global.crypto) {
```

Comment:
Could we add a comment here to clarify this behavior? 

```suggestion
// Assign Node's Crypto to global.crypto if it is not already present
if (!global.crypto) {
```